nengo/probe.py

```diff
@@ -28,9 +30,10 @@ def validate(self, probe, attr):
                                 "into 'weights'. Probe 'weights' instead.",
                                 since="v2.1.0")
         if attr not in probe.obj.probeable:
-            raise ValidationError("Attribute %r is not probeable on %s.\n"
+            raise ValidationError("Attribute %r is not probeable on %s.%s"
```

Comment:
You might be right. Apparently line endings are automatically converted for files read or written in text (as opposed to binary) mode. I assume the `print` uses a file descriptor in text mode, so that it should also convert line endings, but did not verify this.